docker-compose.yml

```
@@ -7,6 +7,10 @@ services:
     depends_on:
       db:
         condition: service_healthy
+    options:
+        loki-url: ${LOKI_URL}
+        loki-external-labels: |
+          job=observer-metrics
```

This is missing the logging section 

```suggestion
    logging:
      driver: loki
      options:
        loki-url: ${LOKI_URL}
        loki-external-labels: |
          job=observer-metrics
```